newIDE/app/src/InstancesEditor/TileSetVisualizer.js

```diff
@@ -664,101 +737,96 @@ const TileSetVisualizer = ({
       <Line justifyContent="stretch" noMargin>
         {atlasResourceName && (
           <div
-            style={styles.tileContainer}
-            ref={tileContainerRef}
-            {...(interactive ? interactionCallbacks : undefined)}
-            {...longTouchProps}
+            style={styles.tilesetAndTooltipsContainer}
+            ref={tilesetAndTooltipContainerRef}
           >
-            <CorsAwareImage
-              style={styles.atlasImage}
-              alt={atlasResourceName}
-              src={ResourcesLoader.getResourceFullUrl(
-                project,
-                atlasResourceName,
-                {}
-              )}
-              onLoad={e => {
-                if (onAtlasImageLoaded)
-                  onAtlasImageLoaded(e, atlasResourceName);
-              }}
-            />
-
-            {interactive && hoveredTile && displayedTileSize && (
-              <Tile
-                key={`hovered-tile`}
-                size={displayedTileSize}
-                x={hoveredTile.x}
-                y={hoveredTile.y}
-                title={getTileIdFromGridCoordinates({
-                  x: hoveredTile.x,
-                  y: hoveredTile.y,
-                  columnCount,
-                }).toString()}
+            <div
+              style={styles.tilesetContainer}
+              ref={tilesetContainerRef}
+              {...(interactive ? interactionCallbacks : undefined)}
+              {...longTouchProps}
+              onMouseMove={onMouseMove}
+              onMouseEnter={onMouseEnter}
+              onMouseLeave={onMouseLeave}
```

Comment:
Could/should this be onPointerXxx?

Reply:
To handle the tooltip display, either you're on desktop and it watches the mouse, either on mobile with the long touch props.
Before that, it was using the `title` attribute of the tile components but it's too slow to appear so I handled it with JS

Reply:
I'm usually not a big fan of making custom tooltips, but let's go ahead with this.